sql/core/src/test/resources/sql-tests/results/describe.sql.out

```diff
@@ -68,67 +68,74 @@ DESC FORMATTED t
+Owner:              	                    	                    
+Created: 
 Last Access: 
 Location: sql/core/spark-warehouse/t	                    
-Owner:              	                    	                    
-Partition Provider: 	Catalog             	                    
-Storage Desc Parameters:	                    	                    
-Table Parameters:   	                    	                    
 Table Type:         	MANAGED             	                    
-a                   	string              	                    
-b                   	int                 	                    
-c                   	string              	                    
-c                   	string              	                    
-d                   	string              	                    
-d                   	string
+Comment:            	table_comment       	                    
+Table Parameters:   	                    	                    
+                    	                    	                    
+# Storage Information	                    	                    
+Compressed:         	No                  	                    
+Storage Desc Parameters:	                    	                    
+Partition Provider: 	Catalog
 
 
 -- !query 6
 DESC EXTENDED t
 -- !query 6 schema
 struct<col_name:string,data_type:string,comment:string>
 -- !query 6 output
-# Detailed Table Information	CatalogTable(
-	Table: `default`.`t`
-	Created: 
-	Last Access: 
-	Type: MANAGED
-	Schema: [StructField(a,StringType,true), StructField(b,IntegerType,true), StructField(c,StringType,true), StructField(d,StringType,true)]
-	Provider: parquet
-	Partition Columns: [`c`, `d`]
-	Comment: table_comment
-	Storage(Location: sql/core/spark-warehouse/t)
-	Partition Provider: Catalog)	                    
-# Partition Information	                    	                    
-# col_name          	data_type           	comment             
 a                   	string              	                    
 b                   	int                 	                    
 c                   	string              	                    
+d                   	string              	                    
+# Partition Information	                    	                    
+# col_name          	data_type           	comment             
 c                   	string              	                    
 d                   	string              	                    
-d                   	string
+                    	                    	                    
+# Detailed Table Information	                    	                    
+Table: `default`.`t`
+Created: 
+Last Access: 
+Type: MANAGED
+Provider: parquet
+Partition Columns: [`c`, `d`]
+Comment: table_comment
+Storage(Location: sql/core/spark-warehouse/t)
+Partition Provider: Catalog
+Schema: root
+ |-- a: string (nullable = true)
+ |-- b: integer (nullable = true)
+ |-- c: string (nullable = true)
+ |-- d: string (nullable = true)
```

Comment:
yea we do not need, but I think `CatalogTable.toString` should still have the schema. Actually why do we have `DESC EXTENDED`? I think `DESC FORMATTED` is good enough